tests/memory/enc/fragment.c

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <openenclave/enclave.h>
+#include <openenclave/internal/globals.h>
+#include <openenclave/internal/tests.h>
+
+#include <malloc.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "memory_t.h"
+
+#define MALLOC_SIZE_SMALL 1024
+#define EFFICIENCY_TEST_TIMES 5000000
+
+#define MALLINFO_FIELD_TYPE size_t
+struct mallinfo
+{
+    MALLINFO_FIELD_TYPE arena;    /* non-mmapped space allocated from system */
```

Comment:
I am wondering why these cant be just of type size_t instead of being of type MALLINFO_FIELD_TYPE